h/scripts/init_elasticsearch.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
```

Comment:
For h there's also an `init_elasticsearch.py` script that follows the same pattern as `init_db.py`.

None of our other apps have anything else to initialize but if they ever do they can follow this pattern of having multiple `init_*.py` scripts.

This isn't used by the tests since it requires a Pyramid config file to load settings from and the tests don't have a config file. It *could* be used by the tests if it was refactored to read settings from environment variables or command line arguments instead of a config file. But the tests already have their own fixtures for initializing Elasticsearch that work fine so :man_shrugging: